GPT/gpt.talon

```diff
@@ -11,6 +11,12 @@ model <user.modelPrompt> [this] [{user.modelInsertionMethod}]$:
     result = user.gpt_apply_prompt(modelPrompt, text)
     user.gpt_insert_response(result, modelInsertionMethod or "")
 
+model paste:
```

Comment:
Perhaps we can think if there is a different way to word this. Since `model paste` might be confusing and is a bit vague. I am not really sure a better alternative though. 

`model combine` is a bit more semantic I feel. But not super happy with that either

Reply:
`model combination paste` is long but is very clear. I would be fine with that honestly.

Reply:
What about "model blend paste"? For the clipboard version and "model blend <cursorless destination >" for the cursorless flavor?

Reply:
Or maybe even just "model blend [target] [destination] where:

1. If no target or destination it blends with clipboard
2. If target and no destination, it blends every target and pastes the result
3. If target and destination then it blends every target and the destination and inserts to the destination

Reply:
I like the latter with `model blend [target] [destination]`. Sounds good to me and seems pretty extensible! 

with that in mind, I think we should rename the `gpt_smart_clipboard` function. Since we aren't really using the clipboard or pasting.

Reply:
Yep, makes sense. I'll change that up shortly.